src/main/java/com/limechain/network/protocol/grandpa/GrandpaEngine.java

```diff
@@ -87,7 +90,7 @@ private void handleCommitMessage(byte[] msg, PeerId peerId) {
         ScaleCodecReader reader = new ScaleCodecReader(msg);
         CommitMessage commitMessage = reader.read(new CommitMessageScaleReader());
         if (isBlockAlreadyReached(commitMessage.getVote().getBlockNumber())) {
-            log.log(Level.INFO, String.format("Received commit message for already reached block %d from peer %s",
+            log.log(Level.INFO, String.format("Received commit message for finalized block %d from peer %s",
```

Comment:
No need to spam the logs with this message IMO. Level.FINE would be more suitable.

Reply:
Updated.